client/app/telecom/telephony/alias/dashboard/telecom-telephony-alias-dashboard.controller.js

```diff
@@ -0,0 +1,180 @@
+angular.module('managerApp').controller('TelecomTelephonyAliasDashboardController', class TelecomTelephonyAliasHomeController {
+  constructor(
+    $state, $stateParams, $translate, $uibModal,
+    ChartjsFactory, OvhApiTelephony, Toast,
+    voipService,
+    TELEPHONY_ALIAS_CONSUMPTION,
+  ) {
+    this.$state = $state;
+    this.$translate = $translate;
+    this.$uibModal = $uibModal;
+    this.ChartjsFactory = ChartjsFactory;
+    this.OvhApiTelephony = OvhApiTelephony;
+    this.Toast = Toast;
+    this.voipService = voipService;
+
+    this.TELEPHONY_ALIAS_CONSUMPTION = TELEPHONY_ALIAS_CONSUMPTION;
+
+    this.billingAccount = $stateParams.billingAccount;
+    this.serviceName = $stateParams.serviceName !== 'default' ? $stateParams.serviceName : null;
+  }
+
+  $onInit() {
+    this.hideDetails = false;
+    this.number = null;
+    this.consumption = {
+      incoming: {},
+      outgoing: {},
+      chart: new this.ChartjsFactory(
+        angular.copy(this.TELEPHONY_ALIAS_CONSUMPTION.chart),
+      ),
+    };
+
+    this.fetchService();
+  }
+
+  fetchService() {
+    this.loading = true;
+    this.voipService.fetchSingleService(this.billingAccount, this.serviceName).then((number) => {
+      if (number) {
```

Comment:
`number`? What number ? A phone number ? What does this variable contains ? The name should convey what it contains.

Also, maybe `_(number).isNumber()`

Reply:
It is a telephony alias, so it will be named this way